### ISSUE TITLE
Newly added modules don't work

### ISSUE THREAD

**Issue by F1RM-xD** [ChatTriggers/ChatTriggers#382]
**Describe the bug**

**/ct import [module name] results in "Unable to import module example" for every module I try. Manually adding modules to the modules file results in ct recognizing the module in game but no commands or features of the modules do anything.**

**To Reproduce**
**Steps explained in description and screenshots.**

**Expected behavior**
**A clear and concise description of what you expected to happen.
I expect modules to work when I add them to my modules file, and for /ct import to add modules.**

**Screenshots**
![image_2023-05-26_151958033](https://github.com/ChatTriggers/ChatTriggers/assets/134739445/29318253-40b1-4b72-98c0-8b1edf71f04e)

![image_2023-05-26_152048602](https://github.com/ChatTriggers/ChatTriggers/assets/134739445/9596d13c-a1eb-4c92-a244-9c6dac9f10f6)

![image_2023-05-26_152057568](https://github.com/ChatTriggers/ChatTriggers/assets/134739445/a67ca0b3-0cb7-46d6-ae2e-49b34d4d4948)

**Setup (please complete the following information):**
 - OS: [e.g. Windows/Mac/Ubuntu]              **Windows**
 - ChatTriggers Version: [e.g. 0.18.4]             **2.2.0**
 - Minecraft Version: [e.g. 1.8.9]                    **1.8.9 (skyclient)**
- Java Version: [e.g. 1.8.0_231]                      **1.8.0_351**

**Additional context**

**Comment by camnwalter** [comment]
To fix modules not importing, refer to the wiki: https://github.com/ChatTriggers/ChatTriggers/wiki. And if you are manually downloading modules, try /ct load, and if that doesn't fix the issues, look in /ct console js for errors.

**Comment by F1RM-xD** [comment]
I have no idea what any of this means. 
![image](https://github.com/ChatTriggers/ChatTriggers/assets/134739445/91973196-bb08-4211-81c8-24873306d2c2)
Could you please tell me how to append the bin\javaw.exe to the path? (to fix import)

**Comment by FalseHonesty** [comment]
Try using these instructions: https://gist.github.com/NoahTheNerd/d1406abedffa38d8b834b21324dea7ef, the ones in the wiki are hard to follow.

**Comment by F1RM-xD** [comment]
Fixed my issues thanks so much!
